src/core/lib/transport/handshaker.cc

```diff
@@ -193,6 +190,7 @@ void HandshakeManager::DoHandshake(grpc_endpoint* endpoint,
       grpc_slice_buffer_swap(args_.read_buffer,
                              &(acceptor->pending_data->data.raw.slice_buffer));
     }
+    event_engine_ = args_.args.GetObjectRef<EventEngine>();
```

Comment:
Suggest moving this down to line 203, right before we actually use it, just so all of the EE-related code is in one place.